lib/src/specs/type_reference.dart

```diff
@@ -39,6 +39,10 @@ abstract class TypeReference extends Expression
   @override
   BuiltList<Reference> get types;
 
+  /// Optional nullability.
```

Comment:
Expand this, "An emitter may ignore this if the output is not targeting a language version that supports null safety."

Reply:
Done.